src/main/kotlin/no/nav/syfo/persistering/db/PersisterDialogmeldingQueries.kt

```diff
@@ -99,6 +100,20 @@ fun Connection.erDialogmeldingOpplysningerLagret(dialogmeldingid: String) =
         }
     }
 
+fun Connection.hentMottattTidspunkt(shaString: String) =
+    use { connection ->
+        connection.prepareStatement(
+            """
+                SELECT o.mottatt_tidspunkt FROM DIALOGMELDINGOPPLYSNINGER o
+                INNER JOIN DIALOGMELDINGDOKUMENT dok ON dok.id = o.id
+                WHERE dok.sha_string=?;
+                """
+        ).use {
+            it.setString(1, shaString)
+            it.executeQuery().toList { getString("mottatt_tidspunkt") }.firstOrNull() ?: "Fant ikke tidspunkt"
```

Comment:
`mottatt_tidspunkt` er Timestamp, ikke string og hentes ut som det. Så konverterer man det typisk til LocalDateTime.

Husker ikke helt hvordan det er i Padm2, men om man vet at en eksisterer en rad med `shaString` man bruker er så skal det eksistere en `mottatt_tidspunkt` og dermed burde en evt. null/tom verdi her være en feil.

Reply:
Ja stemmer, det skal egentlig ikke være mulig å få fallbacken. Kan eventuelt endre til first() i stedet for firstOrNull() slik at den kaster exception dersom det skulle skje.